requirements.txt

```diff
@@ -1,6 +1,8 @@
-Django==3.0.6
+Django==3.0.7
 django-allow-cidr==0.3.1
 gunicorn==20.0.4
 pylint==2.5.2
 pylint-django==2.0.15
-python-dotenv==0.13.0
\ No newline at end of file
+python-dotenv==0.13.0
```

Comment:
if we have a blank line at the end of the file then we will not get these false diffs.